src/SWP/Bundle/ContentBundle/Doctrine/ORM/ArticleAuthorRepository.php

```diff
@@ -74,7 +74,12 @@ private function applyCustomFiltering(Criteria $criteria, QueryBuilder $queryBui
             $criteria->remove($name);
         }
 
+        $properties = array_merge($this->getClassMetadata()->getFieldNames(), $this->getClassMetadata()->getAssociationNames());
         foreach ($criteria as $key => $criterion) {
+            if (!in_array(str_replace('exclude_', '', $key), $properties)) {
```

Comment:
`Using '\in_array(...)' would enable some of opcode optimizations` and `Third parameter should be provided to clarify if type safety is important in this context`